stable/nginx-ingress/templates/nginx-deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: controller
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+  name: {{ template "fullname" . }}-controller
+spec:
+  replicas: {{ .Values.nginx.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        component: controller
+        release: "{{ .Release.Name }}"
+    spec:
+      containers:
```

Comment:
Could you add this as a sidecar to make monitoring easier?
https://github.com/hnlq715/nginx-vts-exporter